docs/templates/python/material/_base/attribute.html

```diff
@@ -0,0 +1,101 @@
+    {% set root_members = False %}
+  {% else %}
+    {% set show_full_path = config.show_object_full_path %}
+  {% endif %}
+
+  {% set attribute_name = attribute.path if show_full_path else attribute.name %}
+
+  {% if not root or config.show_root_heading %}
+    {% filter heading(
+        heading_level,
+        role="data" if attribute.parent.kind.value == "module" else "attr",
+        id=html_id,
+        class="doc doc-heading",
+        toc_label=('<code class="doc-symbol doc-symbol-toc doc-symbol-attribute"></code>&nbsp;'|safe if config.show_symbol_type_toc else '') + attribute.name,
+    ) %}
+
+      {% block heading scoped %}
+        {% if config.show_symbol_type_heading %}<code class="doc-symbol doc-symbol-heading doc-symbol-attribute"></code>{% endif %}
+        {% if config.separate_signature %}
+          <span class="doc doc-object-name doc-attribute-name">{{ attribute_name }}</span>
+        {% else %}
+          {%+ filter highlight(language="python", inline=True) %}
+            {{ attribute_name }}{% if attribute.annotation %}: {{ attribute.annotation }}{% endif %}
+            {% if attribute.value %} = {{ attribute.value }}{% endif %}
+          {% endfilter %}
+        {% endif %}
+      {% endblock heading %}
+
+      {% block labels scoped %}
+        {% with labels = attribute.labels %}
+          {% include "labels.html" with context %}
+        {% endwith %}
+      {% endblock labels %}
+
+    {% endfilter %}
+
+    {% block signature scoped %}
+      {% if config.separate_signature %}
+        {% filter format_attribute(attribute, config.line_length, crossrefs=config.signature_crossrefs) %}
+          {{ attribute.name }}
+        {% endfilter %}
+      {% endif %}
+    {% endblock signature %}
+
+  {% else %}
+
+    {% if config.show_root_toc_entry %}
+      {% filter heading(heading_level,
+          role="data" if attribute.parent.kind.value == "module" else "attr",
+          id=html_id,
+          toc_label=('<code class="doc-symbol doc-symbol-toc doc-symbol-attribute"></code>&nbsp;'|safe if config.show_symbol_type_toc else '') + attribute.name,
+          hidden=True,
+      ) %}
+      {% endfilter %}
+    {% endif %}
+    {% set heading_level = heading_level - 1 %}
+  {% endif %}
+
+  <div class="doc doc-contents {% if root %}first{% endif %}">
+    {% block contents scoped %}
+      {% block docstring scoped %}
+        {% with docstring_sections = attribute.docstring.parsed %}
+          {% include "docstring.html" with context %}
+        {% endwith %}
+      {% endblock docstring %}
+    {% endblock contents %}
+
+    {# START NEW CODE #}
+    {% if pipeline_steps(attribute_name) %}
+    {# https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks #}
+    <details class="success">
+      <summary>Pipeline steps</summary>
+      <p>
+        The following steps are directly affected by changes in
```

Comment:
This preposition sounds "better" to me – WDYT?
```suggestion
        The following steps are directly affected by changes to
```